ArduCopter/mode_acro.cpp

```diff
@@ -24,6 +24,9 @@ void Copter::ModeAcro::run()
     float target_roll, target_pitch, target_yaw;
     float pilot_throttle_scaled;
 
+    // initialize smoothing gain
+    attitude_control->set_smoothing_gain(get_smoothing_gain());
```

Comment:
Interesting, I was expecting this to be in the init method, instead of run (as quadplane does). Is there any advantage to have it here?

Reply:
the set-smoothing-gain just needs to be run somewhat regularly so that if the user has updated the RC_FEEL_RP parameter that it's updated in the attitude controller.  Neither Leonard nor I were very happy with how often this gets called so in the lthall-new-loiter11 branch we've just moved the parameter straight into the AttitudeControl library which completely removes the need for the set-smoothing-gain calls.